### PR TITLE
adds maxlength restriction to pipeline test input/output

### PR DESCRIPTION
**Pull request by esotrope**
image

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -1578,13 +1578,15 @@
...
+          "required": "*",
+          "maxLength": "(Please use a smaller test input)"
         }
       },
       "output": {
         "label": "Expected Output",
         "error": {
-          "required": "*"
+          "required": "*",
+          "maxLength": "(Please use a smaller test output)"
```
Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: this text alright?
maybe "Please reduce the test input size" is better. let me know what you guys think

Comment (gregcharles) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Maybe explicitly reference the acceptable size? It's likely the users would just paste in a real file, even if the admin guide tells them not to do that. Then they'd have to iterate through reduced sizes until one works. (You could have a comment over TextMaximumLength that warns devs to update the error messages if they change the value.)

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: yeah i was thinking it isn't so helpful since the number is so big (65535) but maybe it is?

Comment (gregcharles) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: If it's not done by parameter, then it'd be just as easy to say 64 kilobytes in the messages.

Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: this look ok?
image

**Pull request merged**